Library/Formula/llvm.rb

```diff
@@ -1,3 +1,29 @@
+class CodesignRequirement < Requirement
+  fatal true
+
+  def setup
+    FileUtils.touch "/tmp/llvm_check.txt"
+  end
+
+  satisfy do
+    setup
+    quiet_system "/usr/bin/codesign", "-s", "lldb_codesign", "/tmp/llvm_check.txt"
+  end
```

Comment:
Bingo! That's what I was looking for, but I couldn't remember that `mktemp` was a FileUtils thing :speak_no_evil:.